source/fundamentals/crud.txt

```diff
@@ -28,6 +28,17 @@ Some operations combine aspects of read and write operations. See our
 guide on :doc:`compound operations </fundamentals/crud/compound-operations>`
 to learn more about these hybrid methods.
 
+Compatibility
+-------------
+
+.. |page-topic| replace:: use CRUD operations
```

Comment:
S: I don't think anyone would really be lost by "use" here, but I think of "use" as being something you do with a specific tool (use a hammer, use a driver, etc.) whereas a CRUD operation is more like a series of actions--something you perform or execute

```suggestion
.. |page-topic| replace:: perform CRUD operations
```